src/ui.cpp

```diff
@@ -216,10 +322,10 @@ uilist::uilist( const std::string &msg, std::initializer_list<const char *const>
 
 uilist::~uilist()
 {
-    shared_ptr_fast<ui_adaptor> current_ui = ui.lock();
-    if( current_ui ) {
-        current_ui->reset();
-    }
+    //shared_ptr_fast<uilist_impl> current_ui = ui.lock();
+    //if( current_ui ) {
+    //    current_ui->reset();
+    //}
```

Comment:
Some more commented code. Remove?
```suggestion
```

Reply:
Maybe remove the destructor all together, if possible?

Reply:
Yea. Can’t entirely remove the destructor due to c++ reasons, but it can just be a default destructor.